### DISCUSSION TITLE
`_geo` must be configured as a displayedAttribute to see `_geoDistance`

### DISCUSSION THREAD

**Discussion by LiamKarlMitchell** [meilisearch/product#696] [Feedback & Feature Proposal]
It does not seem possible to have `_geoDistance` visible on its own without the `_geo` information also displayed in the document results.

What if I don't want to include the _geo lat/lng in the response for some reasons?

Is there a way to have only `_geoDistance` displayed but not `_geo`?

Both _geoRadius as a filter and _geoPoint as as sort need to be used to see _geoDistance.
And _geo needs to be displayed.

Request:
```json
{
    "q": "test",
    "filter": "_geoRadius(12.3456, -12.3456, 100000)",
    "sort": [
        "_geoPoint(12.3456, -12.3456):asc"
    ]
}
```

Response:
```json
{
    "hits": [
        {
            "id": 3,
            "name": "Test",
            "location": "Some Place",
            "_geo": {
                "lat": 12.345,
                "lng": -12.345
            },
            "_geoDistance": 310
        }
    ],
    "query": "test",
    "processingTimeMs": 0,
    "limit": 20,
    "offset": 0,
    "estimatedTotalHits": 1
}

**Comment by CaroFG** [comment]
Hi! Indeed, as stated in the [documentation](https://www.meilisearch.com/docs/learn/fine_tuning_results/geosearch#finding-the-distance-between-a-document-and-a-_geopoint) "returned documents will only include _geoDistance if _geo is present in the [displayedAttributes list](https://www.meilisearch.com/docs/learn/configuration/displayed_searchable_attributes)."

As this seems more like a feature request, I am transferring this to the product repository It would be great if you could elaborate on why this is important to you 🙏

**Reply by LiamKarlMitchell** [reply]
Yeah I had made a PR for updating the docs the other night :)
Thanks I"ll elaborate on it some more.

Thankfully for our business case it does not seem a strict requirement.
However having _geo lat/lng on the wire is not ideal as we don't actually need it in the client that queries the search only the distance is of importance best to keep things light weight if not needing it turn it off etc. So more of a nice to have than a need in my current scenario :)

We don't display things on a map in our mobile app that uses Meilisearch so it's not necessary data.

But I could see it being a stricter requirement in some other use-case, e.g. distance to nearby things but your not allowed to know their exact location for some reasons/privacy/anti stalking on a dating app or who knows.

**Reply by macraig** [reply]
Thanks for sharing your use case with us @LiamKarlMitchell ! I agree, it would make a lot of sense where the privacy of the location is essential. I've added it to our list of feature requests and will keep this discussion open in case other users want to upvote or add their own use cases.
